loguru.cpp

```diff
@@ -75,6 +75,9 @@
 
 // TODO: use defined(_POSIX_VERSION) for some of these things?
 
+#ifndef LOGURU_USE_LOCALE
+	#define LOGURU_USE_LOCALE 0
```

Comment:
Why is this defined on two places?

Reply:
Thanks for catching that @bylowerik ; I've knocked it out of the `.cpp` so it's only defined in the `.hpp`.